services/src/main/scala/cromwell/pipeline/service/ProjectConfigurationService.scala

```diff
@@ -39,10 +39,14 @@ class ProjectConfigurationService(repository: ProjectConfigurationRepository, pr
     }
   }
 
+  private def getConfigurationsById(projectId: ProjectId, userId: UserId): Future[Seq[ProjectConfiguration]] =
+    projectService.getUserProjectById(projectId, userId).flatMap(_ => repository.findByProjectId(projectId))
+
   def getConfigurationById(projectId: ProjectId, userId: UserId): Future[Option[ProjectConfiguration]] =
-    projectService
-      .getUserProjectById(projectId, userId)
-      .flatMap(_ => repository.getById(projectId).map(_.filter(_.active)))
+    getConfigurationsById(projectId, userId).map(_.filter(_.active).headOption)
+
+  def getLastConfigurationById(projectId: ProjectId, userId: UserId): Future[Option[ProjectConfiguration]] =
+    getConfigurationsById(projectId, userId).map(_.headOption)
```

Comment:
Why do we need this method? If configuration is not active, then it is invisible for everyone.

Reply:
If we create new configuration, and we have last deactivated, then we still need to create new configuration with incremented number.

Reply:
I chaned this method with filter and sorting